### PR TITLE
Updated Additional RAT account in Demo app.

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
RAT is not working due to the default value set in the PLIST.
Now it is updated, also added another RAT account to send analytics.

## Links
Add links to github/jira issues, design documents and other relevant resources (e.g. previous discussions, platform/tool documentation etc.)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

**Review by khairul-alam-licon — APPROVED** [review]
👍👍

File: Example/Utils/Config.swift [hunk 1]
```diff
@@ -13,12 +13,15 @@ class Config: NSObject {
...
+    // swiftlint:disable:next todo
+    // TODO: Make it as CI Configurable
     class func current() -> MiniAppSdkConfig {
         MiniAppSdkConfig(baseUrl: Config.userDefaults?.string(forKey: Config.Key.endpoint.rawValue),
             rasProjectId: Config.userDefaults?.string(forKey: Config.Key.projectId.rawValue),
             subscriptionKey: Config.userDefaults?.string(forKey: Config.Key.subscriptionKey.rawValue),
             hostAppVersion: Config.userDefaults?.string(forKey: Config.Key.version.rawValue),
-            isPreviewMode: Config.userDefaults?.bool(forKey: Config.Key.isPreviewMode.rawValue))
+            isPreviewMode: Config.userDefaults?.bool(forKey: Config.Key.isPreviewMode.rawValue),
+            analyticsConfigList: [MAAnalyticsConfig(acc: "477", aid: "998")])
```
Comment (munir-rakuten) [inline Example/Utils/Config.swift:24]: `acc` `aid` values can be stored in a global variable or env file, just a suggestion 🙂

**Review by munir-rakuten — APPROVED** [review]
Looks good to me  👍

File: Example/Utils/Config.swift (hunk 1 above)
Comment (rleojoseph) [reply Example/Utils/Config.swift:24]: @munir-rakuten , this is a quick fix. Also, we are planning to make changes to this by next week (from Settings). So, I felt it is ok for now

**Review by corycaywood — APPROVED** [review]

File: Example/Utils/Config.swift (hunk 1 above)
Comment (donnie-jp) [reply Example/Utils/Config.swift:24]: acc and aid values are internal values that we shouldn't be exposing in source code. That was why I made them configurable from the plist

Comment (rleojoseph) [reply Example/Utils/Config.swift:24]: @donnie-jp . We have the default details hardcoded already in the SDK's code already https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/core/MiniAppAnalytics.swift#L52. Thought this was ok with the additional account, do you want me to remove it immediately?

Comment (donnie-jp) [reply Example/Utils/Config.swift:24]: I guess it's up to your team / @corycaywood. Just flagging the issue cos I happened to see the PR title on bitrise.

Comment (corycaywood) [reply Example/Utils/Config.swift:24]: @donnie-jp We will later be replacing these values with a setting in the Demo App UI.

As for the `1553` setting used by the SDK: this value needs to always be used by the SDK and we don't want the Host App to be able to change it. So as far as I understand, there isn't a way to set it from an environment variable.

**Pull request merged**